src/main/java/SpringJPA/PageController.java

```diff
@@ -69,6 +72,7 @@ public String user(Model model, Principal principal) {
     public String admin(Model model, Principal principal) {
         modifyNavBar(model, principal);
         model.addAttribute("users", userRepository.findAll());
+        model.addAttribute("userApi", new User());
```

Comment:
What does this do exactly?

Reply:
Adds a User object called userApi to the model, it's necessary in order for the front end to make calls to the page controller.